installing/installing_openstack/installing-openstack-installer-custom.adoc

```diff
@@ -30,6 +30,7 @@ include::modules/installation-osp-verifying-external-network.adoc[leveloffset=+1
 include::modules/installation-osp-describing-cloud-parameters.adoc[leveloffset=+1]
 include::modules/installation-obtaining-installer.adoc[leveloffset=+1]
 include::modules/installation-initializing.adoc[leveloffset=+1]
+include::modules/installation-configure-proxy.adoc[leveloffset=+2]
```

Comment:
You also need to update the module to list these assemblies as places that they're used. Otherwise, this seems reasonable.

Reply:
@kalexand-rh - please verify. I just added it.

Reply:
LGTM!